src/screens/track-screen/TrackScreen.tsx

```diff
@@ -53,22 +58,76 @@ const createStyles = (themeColors: ThemeColors) =>
     }
   })
 
+type TrackScreenMainContentProps = {
+  lineup: LineupState<Track>
+  track: Track
+  user: User
+}
+
 /**
- * `TrackScreen` displays a single track and a Lineup of more tracks by the artist
+ * `TrackScreenMainContent` includes everything above the Lineup
  */
-export const TrackScreen = ({ route, navigation }: Props) => {
+const TrackScreenMainContent = ({
+  lineup,
+  track,
+  user
+}: TrackScreenMainContentProps) => {
   const styles = useThemedStyles(createStyles)
+  const pushRouteWeb = usePushRouteWeb()
 
-  const dispatchWeb = useDispatchWeb()
-  const track = useSelectorWeb(getTrack)
-  const user = useSelectorWeb(getUser)
   const currentUserId = useSelectorWeb(getUserId)
-  const moreByArtistLineup = useSelectorWeb(getMoreByArtistLineup, isEqual)
 
   const remixParentTrackId = track?.remix_of?.tracks?.[0]?.parent_track_id
+  const remixTrackIds = track?._remixes?.map(({ track_id }) => track_id) ?? null
   const showMoreByArtistTitle =
-    (remixParentTrackId && moreByArtistLineup.entries.length > 2) ||
-    (!remixParentTrackId && moreByArtistLineup.entries.length > 1)
+    (remixParentTrackId && lineup.entries.length > 2) ||
+    (!remixParentTrackId && lineup.entries.length > 1)
+
+  const moreByArtistTitle = showMoreByArtistTitle && (
+    <Text
+      style={styles.lineupHeader}
+      weight='bold'
+    >{`${messages.moreBy} ${user?.name}`}</Text>
+  )
+
+  const goToAllRemixes = () => {
+    // TODO: sk - implement this page in RN
+    pushRouteWeb(trackRemixesPage(track.permalink))
+  }
+
+  return (
+    <View style={styles.root}>
+      <View style={styles.headerContainer}>
+        <TrackScreenHeader
+          track={track}
+          user={user}
+          uid={lineup?.entries?.[0]?.uid}
+          currentUserId={currentUserId}
+        />
+      </View>
+
+      {track.field_visibility?.remixes &&
```

Comment:
we doing the ? : style now btw?

Reply:
We should yeah, will update. This is all copy & paste stuff

We need to update the style guide too, after discussing w/ the rest of team